_includes/proposals.html

```diff
@@ -3,6 +3,10 @@
   <h2 class="section-headline text-end">State of Proposals</h2>
   <div class="featurelist">
     <h3>Stage 3</h3>
+    <i>
+      Proposals that have been reviewed, implemented in at least one vendor, and are being considered as
```

Comment:
this is incorrect; zero implementations are required for stage 3.

Reply:
On the other hand, "complete spec text" is [listed](https://tc39.github.io/process-document/) as a requirement.